lib/bootstrap.dart

```diff
@@ -146,6 +147,11 @@ Future<void> bootstrap(FutureOr<Widget> Function() builder) async {
               hiveRepository: getIt(),
             ),
           ),
+          BlocProvider<SearchCubit>(
+            create: (context) => SearchCubit(
+              getIt<DBRepository>(),
```

Comment:
Uniformity.

We have named parameters on all the other cubits. Please inject as a named parameter and have it as `getIt()` just like everything else.

Reply:
Fixed